dev/integration_tests/android_semantics_testing/test_driver/main_test.dart

```diff
@@ -63,6 +63,34 @@ void main() {
         await driver.tap(find.text(textFieldRoute));
         // Delay for TalkBack to update focus as of November 2019 with Pixel 3 and Android API 28
         await Future<void>.delayed(const Duration(milliseconds: 500));
+
+        // The text selection menu and related semantics vary depending on if the
+        // clipboard contents are pasteable. Copy some text into the clipboard to
+        // make sure these tests always run with pasteable content in the clipboard.
```

Comment:
Maybe add that ideally we would want to run the tests with an empty clipboard as well, but there is no reliable way to clear the clipboard.